architecture/minimum_viable_product.md

```diff
@@ -0,0 +1,41 @@
+# Minimum Viable Product.
+The mvp should be so simple as it should be. There are so many alternatives.
+In my idea, The minimum viable product is as follows
+
+## Server
+Backend server should be capable of recieving a request with current location from end user. Response of the request is list of auto rickshaws around current location. 
+
+Another facility needed is  registering auto rickshaw cooperative unions and auto rickshaw drivers.
+Drivers will register under a cooperative union and cooperative union will verify the driver.
+Driver rating is another capability needed for MVP.
+
+In some areas, where fare meters is not practical, like villages, it is not possible. 
+There, a commonly agreed fare by the drivers cooperative society is getting paid.
+We can use that and give available to the users. 
+
+People who dont have a smart phone could be able to register in the server, manually by the cooperative society with the basic details.
+
+## Mobile End User Client
+The Android mobile client should be capable of
+* requesting nearby rickshaws with user's current location 
+* process its response and display list of auto rickshaws available.
+* Detailed view of a single auto rickshaw will have the following
+    * Auto Rickshaw Photo
+    * Driver Name
+    * Driver phone number for calling.
+    * facilities availiable such as music, fare meter etc.
```

Comment:
we cant make this mandatory. but we can introduce as a feature. a feature for filtering in search. we can use the commonly agreed fare chart in the future for automation purpose.